scipy/interpolate/_rbfinterp.py

```diff
@@ -387,6 +387,64 @@ def __init__(self, y, d,
         self.epsilon = epsilon
         self.powers = powers
 
+    def _chunk_evaluator(
+            self,
+            x,
+            y,
+            shift,
+            scale,
+            coeffs,
+            memory_budget=None
+    ):
+        """
+        Evaluate the interpolation why controlling memory consumption.
+        We chunk the input if we need more memory than specified.
+
+        Parameters
+        ----------
+        x : (Q, N) float ndarray
+            array of points on which to evaluate
+        y: (P, N) float ndarray
+            array of points on which we know function values
+        shift: (N, ) array
+            Domain shift used to create the polynomial matrix.
+        scale : (N,) float ndarray
+            Domain scaling used to create the polynomial matrix.
+        coeffs: (Q, P+R) float ndarray
```

Comment:
The shape of coeffs is (P + R, S), where R is the number of monomials and S is the number of data dimensions.

Reply:
fixed